docs/tempostack/object_storage.md

```diff
@@ -31,20 +66,20 @@ Tempo Operator supports [Minio](https://min.io/) for TempoStack object storage.
 
     ```console
     kubectl create secret generic tempostack-dev-minio \
-      --from-literal=bucket="<BUCKET_NAME>" \
+      --from-literal=bucketnames="<BUCKET_NAME>" \
```

Comment:
why changing it from `bucket` to `bucketnames`? Seems internally its also called `bucket`. And in case we change it, wouldn't it `bucketname` instead?